wss/1.0/modules/ROOT/partials/transformation-note.adoc

```diff
@@ -0,0 +1,4 @@
+[NOTE]
+--
+Transform operations don't take into account the cryptographic signatures and can end up modifying them.
```

Comment:
"end up"?
Perhaps something like this? "Transform operations ... and might change them."
But now it seems as though the reader needs some explanation of the sorts of changes that transform operations might make to cryptographic signatures.

Reply:
Any transformation to the content can break the cryptographic signatures. It is not obvious to users, but DataWeave doesn't just output the input as-is. It will parse the input, convert to an internal format, and generate a new output. For other use cases it might be indistinguishably from the input for XML processing purposes. Just doing #[payload] for example emits a new output, that can change the formatting and break the signature validations.